airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.tsx

```diff
@@ -0,0 +1,26 @@
+import classnames from "classnames";
+import React from "react";
+
+import styles from "./NumberBadge.module.scss";
+
+interface NumberBadgeProps {
+  value: number;
+  color?: "green" | "red" | "blue" | "default";
+  className?: string;
+  "aria-label"?: string;
+}
+
+export const NumberBadge: React.FC<NumberBadgeProps> = ({ value, color, className, "aria-label": ariaLabel }) => {
+  const numberBadgeClassnames = classnames(styles.circle, className, {
+    [styles.default]: !color,
```

Comment:
```suggestion
    [styles.default]: !color || color === "default",
```

Reply:
So that the style will also apply when a user explicitaly set `default` as a property value.